flang/test/Driver/omp-driver-offload.f90

```diff
@@ -201,3 +201,24 @@
 ! RUN:      -nogpulibc %s 2>&1 \
 ! RUN:   | FileCheck --check-prefix=NO-LIBC-GPU-AMDGPU %s
 ! NO-LIBC-GPU-AMDGPU-NOT: "-lcgpu-amdgpu"
+
+! RUN:   rm -rf %t/Inputs
+
+! RUN:   not %flang -### -v --target=x86_64-unknown-linux-gnu -fopenmp  \
+! RUN:      --offload-arch=gfx900 \
+! RUN:      --rocm-path=%t/Inputs/rocm %s 2>&1 \
```

Comment:
Where does this live? There's one in `clang` but not `flang`.

Reply:
I'm assuming you mean `%S` as well.

Reply:
This test checks if flang reports an error if ROCm path doesn't exist. The next test checks positive scenario.

Reply:
Even in that case I don't think `%t` is right, should probably just point it to some non-existant path off the current directory `%S`.